File: test/fixtures/output/r/httr/application-form-encoded.r
library(httr)

url <- "http://mockbin.com/har"

payload <- "foo=bar&hello=world&hello%20world=world%20hello"

encode <- "form"

response <- VERB("POST", url, body = payload, content_type("application/x-www-form-urlencoded"), encode = encode)

content(response, "text")

File: test/fixtures/output/python/requests/application-form-encoded.py
import requests

url = "http://mockbin.com/har"

payload = "foo=bar&hello=world&hello%20world=world%20hello"
headers = {'content-type': 'application/x-www-form-urlencoded'}

response = requests.request("POST", url, data=payload, headers=headers)

print(response.text)
